From: Jiri Slaby <jslaby@suse.cz>
Subject: x86/PCI: pci, fix section mismatch
References: fate#311968
Patch-Mainline: yes
Git-commit: 73cd3b43f08cc9a9bcb168994b8e9ebd983ff573


Signed-off-by: Thomas Renninger <trenn@suse.de>

pcibios_scan_specific_bus calls pci_scan_bus_on_node which is
__devinit. Mark pcibios_scan_specific_bus __devinit as well since
all users are now __init or __devinit.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
Signed-off-by: Jesse Barnes <jbarnes@virtuousgeek.org>

---
 arch/x86/pci/legacy.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.32-SLE11-SP1/arch/x86/pci/legacy.c
===================================================================
--- linux-2.6.32-SLE11-SP1.orig/arch/x86/pci/legacy.c
+++ linux-2.6.32-SLE11-SP1/arch/x86/pci/legacy.c
@@ -39,7 +39,7 @@ static int __init pci_legacy_init(void)
 	return 0;
 }
 
-void pcibios_scan_specific_bus(int busn)
+void __devinit pcibios_scan_specific_bus(int busn)
 {
 	int devfn;
 	long node;
